CondFormats/JetMETObjects/src/JetResolutionObject.cc

```diff
@@ -0,0 +1,437 @@
+#include <CondFormats/JetMETObjects/interface/JetResolutionObject.h>
+#include <CondFormats/JetMETObjects/interface/Utilities.h>
+#include <FWCore/Framework/interface/EventSetup.h>
+#include <FWCore/Framework/interface/ESHandle.h>
+#include <FWCore/Utilities/interface/EDMException.h>
+
+#include <CondFormats/DataRecord/interface/JetResolutionRcd.h>
+#include <CondFormats/DataRecord/interface/JetResolutionScaleFactorRcd.h>
+
```

Comment:
This is actually needed for the helper functions `JetResolution::get()` and `JetResolutionScaleFactor::get()`